grails-app/migrations/0.9.x/changelog-2023-11-06-1330-drop-nullable-constraint-on-name-column-for-role-table.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+
+  <changeSet author="drodzewicz" id="061120231330-0">
+    <preConditions onFail="MARK_RAN" onError="CONTINUE">
+      <sqlCheck expectedResult="YES">
+        SELECT IS_NULLABLE
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name="role"
+        AND column_name="name"
+        AND TABLE_SCHEMA = DATABASE();
+      </sqlCheck>
+    </preConditions>
+    <addNotNullConstraint tableName="role" columnName="name" defaultNullValue="" columnDataType="VARCHAR(255)" />
+  </changeSet>
+</databaseChangeLog>
```

Comment:
why is the file named "drop nullable constraint" if you add it here?
```xml
 <addNotNullConstraint tableName="role" columnName="name" defaultNullValue="" columnDataType="VARCHAR(255)" />
```

Reply:
When creating the changelog file I thought that `nullable: false` was a separate constraint that I would have to drop, so in a way it does makes sense but I do agree that it would be better to rename it to something like 
`changelog-2023-11-06-1330-add-not-nullable-constraint-on-name-column-for-role-table.xml`

thanks for pointing that out :+1: